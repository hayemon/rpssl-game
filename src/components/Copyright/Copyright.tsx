import React from 'react'

const Copyright: React.FC = () => {
  return (
    <span className='copyright'>
      <a href='http://www.freepik.com'>Designed by pikisuperstar / Freepik</a>
    </span>
  )
}

export default Copyright
