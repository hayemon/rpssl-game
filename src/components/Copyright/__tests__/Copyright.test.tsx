import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import Copyright from '../Copyright'

describe('Copyright', () => {
  it('renders', () => {
    const { asFragment } = render(<Copyright />)
    expect(asFragment()).toMatchSnapshot()
  })
})
