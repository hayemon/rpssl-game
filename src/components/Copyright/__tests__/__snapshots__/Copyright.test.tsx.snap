// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright renders 1`] = `
<DocumentFragment>
  <span
    class="copyright"
  >
    <a
      href="http://www.freepik.com"
    >
      Designed by pikisuperstar / Freepik
    </a>
  </span>
</DocumentFragment>
`;
